src/components/MainMenu/MainMenu.tsx

```diff
@@ -5,77 +5,80 @@ import { Link } from "react-router-dom";
 import ReactSVG from "react-svg";
 
 import { MenuDropdown } from "..";
-import { generateCategoryUrl } from "../../core/utils";
+import { maybe } from "../../core/utils";
 import { baseUrl } from "../App/routes";
 import { CartContext } from "../CartProvider/context";
-import { Error } from "../Error";
+import Offline from "../Offline";
+import Online from "../Online";
 import { OverlayContext, OverlayTheme, OverlayType } from "../Overlay/context";
 import { UserContext } from "../User/context";
-import { GET_MAIN_MENU } from "./queries";
+import NavDropdown from "./NavDropdown";
+import { TypedMainMenuQuery } from "./queries";
 
 import { mediumScreen, smallScreen } from "../App/scss/variables.scss";
-import Offline from "../Offline";
-import Online from "../Online";
 import "./scss/index.scss";
 
+const Svg = (fileName, props?) => (
```

Comment:
I don't think this is a good approach. Wouldn't it be better if we `require()` files to consts here and then use it as `<ReactSVG path={hamburger} className="main-menu__hamburger" />`?